chainer/functions/loss/sigmoid_cross_entropy.py

```diff
@@ -30,50 +25,39 @@ def forward(self, inputs):
         xp = cuda.get_array_module(*inputs)
         x, t = inputs
         self.ignore_mask = (t != self.ignore_label)
-        if self.normalize:
-            count = xp.maximum(1, self.ignore_mask.sum())
-        else:
-            count = max(1, len(x))
-        self.count = count
         # stable computation of the cross entropy.
-        loss = -xp.sum(
-            self.ignore_mask * (x * (t - (x >= 0)) -
-                                xp.log1p(xp.exp(-xp.abs(x)))))
-        return utils.force_array(xp.divide(loss, self.count, dtype=x.dtype)),
+        loss = -(
+            self.ignore_mask *
+            (x * (t - (x >= 0)) - xp.log1p(xp.exp(-xp.abs(x)))))
+
+        return utils.force_array(loss.astype(x.dtype)),
```

Comment:
I added those tests. In order to support scalar input, I kept `utils.force_array`.